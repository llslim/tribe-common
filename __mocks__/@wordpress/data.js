/**
 * External dependencies
 */
import { noop } from 'lodash';

export const select = noop;
