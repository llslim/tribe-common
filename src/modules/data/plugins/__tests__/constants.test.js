/**
 * Internal dependencies
 */
import { constants } from '@moderntribe/common/data/plugins';

describe( 'Plugin constants', () => {
	test( 'Events plugin', () => {
		expect( constants.EVENTS_PLUGIN ).toMatchSnapshot();
	} );
	test( 'Events pro plugin', () => {
		expect( constants.EVENTS_PRO_PLUGIN ).toMatchSnapshot();
	} );
	test( 'Events virtual plugin', () => {
		expect( constants.EVENTS_VIRTUAL_PLUGIN ).toMatchSnapshot();
	} );
	test( 'Tickets plugin', () => {
		expect( constants.TICKETS ).toMatchSnapshot();
	} );
	test( 'Tickets plus plugin', () => {
		expect( constants.TICKETS_PLUS ).toMatchSnapshot();
	} );
} );
