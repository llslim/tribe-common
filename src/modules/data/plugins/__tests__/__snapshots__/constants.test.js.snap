// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plugin constants Events plugin 1`] = `"events"`;

exports[`Plugin constants Events pro plugin 1`] = `"events-pro"`;

exports[`Plugin constants Events virtual plugin 1`] = `"events-virtual"`;

exports[`Plugin constants Tickets plugin 1`] = `"tickets"`;

exports[`Plugin constants Tickets plus plugin 1`] = `"tickets-plus"`;
